tools/train_net.py

```diff
@@ -176,6 +176,11 @@ def main():
         logger.info(config_str)
     logger.info("Running with config:\n{}".format(cfg))
 
+    output_config_path = "{}/config.yml".format(cfg.OUTPUT_DIR)
```

Comment:
`os.path.join` is more robust wrt OS (Windows)

Reply:
just resolved this, should be ready to merge!